apps/next/src/components/RecipientSelect/RecipientSelect.tsx

```diff
@@ -45,7 +47,12 @@ export const RecipientSelect: FC<RecipientSelectProps> = ({
       query={query}
       onQueryChange={onQueryChange}
       value={value}
-      onValueChange={onValueChange}
+      onValueChange={(recipient) => {
+        if (recipient && getType(recipient) === 'contact') {
+          capture('SendContactSelected', { contactSource: 'contacts' });
```

Comment:
Legacy app uses `addressBook` for contacts. Is the change intended?

https://github.com/ava-labs/core-extension/blob/2695c6a9b90ac657d99acde06fc41e71a5d982b2/apps/legacy/src/pages/Send/components/ContactSelect.tsx#L214

Also what about other `contactSource`s? At the moment it could be any of `unknown` / `addressBook` or `accounts` (`recent` recipients not implemented at the moment and also removed from legacy).